### ISSUE TITLE
unsupported operand type(s) for *: 'int' and 'NoneType' when using --remap

### ISSUE THREAD

**Issue by phdoerfler** [migvel/color_trace#5]
```
python3 color_trace_multi.py -v -i page.bmp -r remap.png -o out.svg
```
yields:
`unsupported operand type(s) for *: 'int' and 'NoneType'`.

**Comment by scribblemaniac** [comment]
I can confirm this, full trace:

```
Traceback (most recent call last):
  File "color_trace_multi.py", line 886, in main
    color_trace_multi(inputs, outputs, colors, processcount, dither=dither, **color_trace_kwargs)
  File "color_trace_multi.py", line 780, in color_trace_multi
    total = multiprocessing.Value('i', len(layers) * colors)
TypeError: unsupported operand type(s) for *: 'int' and 'NoneType'
```

**Comment by scribblemaniac** [comment]
This should be fixed when #6 is merged. If you do not want to wait until then, you can pull the code from [my branch](https://github.com/scribblemaniac/color_trace/tree/bugfix).

**Comment by phdoerfler** [comment]
thanks for providing a fix so quickly! I can confirm it fixes the issue.
